#request
uint8 payload_index
uint8 exposure_mode
uint8 iso_data
---
#response
bool result